### PR TITLE
Update README to include backend authentication-related environment v…

### PR DESCRIPTION
**Pull request by nas-tabchiche**
…ariables

### CODE REVIEW

**Review by ab-smith — CHANGES_REQUESTED** [review]
The md formatter breaks the call-out

File: README.md [hunk 1]
```diff
@@ -115,8 +115,7 @@ Check out the online documentation on https://intuitem.gitbook.io/ciso-assistant
 
 <br/>
 
-> [!NOTE]
-> `*` These frameworks require an extra manual step of getting the latest Excel sheet through their website as their license prevent direct usage.
+> [!NOTE] > `*` These frameworks require an extra manual step of getting the latest Excel sheet through their website as their license prevent direct usage.
```
Comment (ab-smith) [inline README.md]: The md formatter breaks the call-out

**Review by eric-intuitem — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (nas-tabchiche) [reply README.md]: should be good now

**Review by ab-smith — APPROVED** [review]

**Pull request merged**